app/src/test/java/com/flatstack/android/login/LoginMapperTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.flatstack.android.login
+
+import com.flatstack.android.graphql.mutation.LoginMutation
+import com.flatstack.android.model.entities.Session
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertEquals
+
+internal class LoginMapperTest {
+    @Test
+    fun mapLogin() {
+        val expectedAccessToken = "access_token"
+        val expectedSignin = LoginMutation.Signin(accessToken = expectedAccessToken)
+        val expectedSession = Session(accessToken = expectedAccessToken)
+
+        assertEquals(LoginMapper().mapLogin(expectedSignin), expectedSession)
```

Comment:
Please split test into 3 sections to make obvious what method is under the test.

Reply:
```
val actualSession = loginMapper.mapLogin(...)

assertThat(actualSession).isEqualTo(expectedSession)
```